assets/src/application/ui/layout/entityList/filterBar/Collapsible.tsx

```diff
@@ -2,42 +2,25 @@ import React, { useRef } from 'react';
 import { useSpring, animated } from 'react-spring';
 
 import { CollapsibleProps } from './types';
-import { SearchInput } from '../../../input';
 import useRect from '../../../../services/hooks/useRect';
 
 /**
  * Collapsible
  * This component is an extraction from EntityListFilterBar
  * and is intended to be used only in context with that parent component.
  */
-const Collapsible: React.FC<CollapsibleProps> = ({
-	entityFilters,
-	listId,
-	searchText,
-	setSearchText,
-	showEntityFilters = false,
-}) => {
+const Collapsible: React.FC<CollapsibleProps> = ({ children, show = false }) => {
 	const ref = useRef<HTMLDivElement>();
 	const { height } = useRect(ref);
 	const props = useSpring({
-		height: showEntityFilters ? height : 0,
-		opacity: showEntityFilters ? 1 : 0,
+		height: show ? height : 0,
+		opacity: show ? 1 : 0,
```

Comment:
nice adherence to DRY

beyond the scope of this PR, but we could likely extract this into its own component for use outside of the filter bar